src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.js

```diff
@@ -59,36 +59,35 @@ export class SetActions extends Component {
               <Option text="Un-Published" value="unpublished" />
             </Select>
 
-            <Button
-              title="Create Item"
-              kind="secondary"
-              id="AddItemButton"
-              className={cx(styles.Action, styles.Create)}
-            >
-              <AppLink
-                className={styles.AppLink}
-                to={`/content/${this.props.modelZUID}/new`}
+            <AppLink to={`/content/${this.props.modelZUID}/new`}>
```

Comment:
Why did the Button > Link structure get inverted?

Reply:
The inverted way was messing with button styles plus this method doesn't follow our convention through out our app. We typically wrap Link > button